akamai-edgekv

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env node
+
+// stub file to kick off compiled app
+// top level CLI needs the file to be started with akamai
+require('./bin/src/edgekv/ekvcommand.js');
```

Comment:
Why would need the src directory inside bin?

Reply:
Since the index.ts is outside the src directory, it creates the src folder inside bin. When I tested it without index.ts it does not create src folder.